package.json

```
@@ -11,7 +11,8 @@
     "typegen": "tsc ./src/transformers.js --outDir dist/types --allowJs --declaration --emitDeclarationOnly",
     "dev": "webpack serve --no-client-overlay",
     "test": "node tests/index.js",
-    "build": "npm run webpack && npm run typegen"
+    "build": "npm run webpack && npm run typegen",
+    "docs": "npx jsdoc-to-markdown --files ./src/**.js"
```

1) I was running into caching issues, it just wouldn't update (`--clear` clears the cache).
2) Creating `docs.md`, so the console isn't spammed

```suggestion
    "docs": "npx jsdoc-to-markdown --clear && npx jsdoc-to-markdown --files ./src/**.js > docs.md"
```